tutorials/neurips_challenge_quickstart.md

```diff
@@ -0,0 +1,201 @@
+# NeurIPS 2023 LLM Efficiency Challenge Quickstart Guide
+
+
+
+The [NeurIPS 2023 Efficiency Challenge](https://llm-efficiency-challenge.github.io/) is a competition focused on training **1 LLM for 24 hours on 1 GPU** – the team with the best LLM gets to present their results at NeurIPS 2023.
+
+This quick start guide is a short starter guide illustrating the main steps to get started with Lit-GPT, which was selected as the competition's official starter kit. 
+
+
+
+&nbsp;
+
+## Competition Facts
+
+**Permitted GPUs:**
+
+- 1x A100 (40 GB RAM);
+- 1x RTX 4090 (24 GB RAM).
+
+**Permitted models:**
+
+- All transformer-based LLM base models that are not finetuned yet.
+
+The subset of Lit-GPT models supported in this competition is listed in the Table below.
+
+| Models in Lit-GPT         | Reference                                                    |
+| ------------------------- | ------------------------------------------------------------ |
+| Meta AI Llama 2 Base      | [Touvron et al. 2023](https://arxiv.org/abs/2307.09288)      |
+| TII UAE Falcon Base       | [TII 2023](https://falconllm.tii.ae/)     
+| OpenLM Research OpenLLaMA | [Geng & Liu 2023](https://github.com/openlm-research/open_llama) |
+| EleutherAI Pythia         | [Biderman et al. 2023](https://arxiv.org/abs/2304.01373)     |
+| StabilityAI StableLM Base | [Stability AI 2023](https://github.com/Stability-AI/StableLM) |
+
+**Permitted datasets**
+
+Any open-source dataset is allowed. However, [per competition rules](https://llm-efficiency-challenge.github.io/challenge), datasets that utilize "generated content" from other LLMs are not permitted.
+
+Examples of permitted datasets are the following:
+
+- [Databricks-Dolly-15](https://huggingface.co/datasets/databricks/databricks-dolly-15k)
+- [OpenAssistant Conversations Dataset (oasst1)](https://huggingface.co/datasets/OpenAssistant/oasst1)
+- [The Flan Collection](https://github.com/google-research/FLAN/tree/main/flan/v2)
+
+It is allowed to create your own datasets if it is made publicly accessible under an open-source license.
```

Comment:
```suggestion
You are allowed to create your own datasets if they are made publicly accessible under an open-source license, and they are not generated from other LLMs (even open-source ones).
```

Reply:
That's a bad one, somehow must have missed it when proofreading. Thanks